CHANGELOG.md

```diff
@@ -0,0 +1,1104 @@
+sunnypilot Version 2025.001.000 (2025-10-25)
+========================
+* Major rewrite
+  * Most features are intended to be identical to previous versions with slight improvements
+  * Fully adopts upstream commaai’s openpilot, opendbc (car interface and safety), and panda test suites to ensure consistent safety compliance and reliability across all systems
+  * Added regression testing to verify expected behavior and maintain stability across core modules
+  * Aligns with comma.ai’s safety policy: preserving driver monitoring, actuation checks, and safety test suite coverage
+  * Some features have not yet been reimplemented in this rewrite and are temporarily disabled in this release. They will return in a future version once fully ported and validated:
+    * Visuals: Rocket Fuel
+    * Visuals: Displaying Braking Status
+    * Vehicle: Toyota - Enforce Stock Longitudinal Control
+    * Subaru: Increase Steering Torque
+    * Longitudinal: Acceleration Personality
+    * UI: Display CPU Temperature on Sidebar
+    * Lateral: Block Lane Change with Road Edge Detection
+    * UI: Display DM Camera in Reverse Gear
+    * UI: Auto-hide Selected UI Elements
```

Comment:
We do allow hiding the speedometer, not identical to how it was but it is something (?) We won't tackle this until after raylib